yamscripts.yml

```diff
@@ -58,7 +59,7 @@ scripts:
 
   testmc:
     lint: =>lint
-    build: =>bundle:webpack && =>bundle:locales
+    build: =>bundle:eslint-fix && =>bundle:webpack && =>bundle:locales
```

Comment:
This runs lint before running fix, so `npm test` / `npm run testmc` will fail before fixing. Separately, travis does `npm test` which would fix up somebody's bad PR that didn't prettier before push.

Reply:
Splitting apart as suggested by Mardak, so this issue is no longer a thing.